aim/storage/treeutils_.pyx

```diff
@@ -12,6 +12,28 @@ from aim.storage.object import CustomObjectBase, CustomObject
 from aim.storage.treeview import TreeView
 from aim.storage.inmemorytreeview import InMemoryTreeView
 
+try:
+    from omegaconf import OmegaConf, Container
+except ModuleNotFoundError:
+    omegaconf_installed = False
+else:
+    omegaconf_installed = True
+
+
+def encode_non_native_object(
```

Comment:
lets keep the `treeutils` generic.
instead we can add a `CustomObject` which will wrap OmegaConf Container and use its' dict-like interface to serialize the data.
@mahnerak thoughts/suggestions?

Reply:
As discussed earlier - non native converter is moved to another file